### ISSUE TITLE
Markdown rendering too frequent

### ISSUE THREAD

**Issue by yadayadaydadaa** [microsoft/vscode#138669]
Reopening #104649. The (slow) rendering of the markdown preview is way too frequent, even more frequent than the rendering speed on larger documents. The only workaround, to avoid getting a headache from really nasty document jumping on larger documents and with extensions that provide math support, is to disable `markdown.preview.scrollPreviewWithEditor`. According to https://github.com/microsoft/vscode/issues/104649#issuecomment-864488948 the rendering frequency is hard-coded in vscode to 300ms so this is *not out of scope* unlike the idiot bot that closed the issue claims it to be. The rendering frequency should be configurable, and better yet, the *rendering should not happen until typing has stopped*.

Oh well, what am I typing, shouting into the wind. I expect the idiot bot to flash-close this issue as well, because developers don't give a shit about anything these days.

**Comment by mjbvz** [comment]
@yadayadaydadaa Your comments are unnecessarily rude, which is not productive and makes you look bad

I closed #104649 because it seems like a workaround for the core problem. Manually having to configure timeouts is not a good experience (plus I believe  `markdown all in one` provides its own markdown preview engine so this setting would not even help them)

The incremental updating added by #136255 is a better solution

**Comment by yzhang-gh** [comment]
Hi @mjbvz, I'm the author of Markdown All in One (don't think it is a good name now 🤣).
Thanks for all your effort on the Markdown support in VS Code.

Markdown All in One doesn't provide a new preview but contributes to the built-in one (so that it is still compatible with other Markdown extensions like [Markdown Footnotes](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-footnotes))

```js
md.use(require("markdown-it-task-lists"));
md.use(require("@neilsustc/markdown-it-katex"), katexOptions);  // <- math here
```

(While [Markdown Preview Enhanced](https://marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced) does provide its own preview.)

> I closed #104649 because it seems like a workaround for the core problem.

For Markdown All in One, I don't think it can provide a solution for this issue.

(Personally I think #104649 should not be locked as I do see a few people complaining about it, at least allow people to comment.)
